xml/Home.xml

```diff
@@ -169,7 +169,7 @@
 					</include>
 					<control type="grouplist" id="7001">
 						<include>WidgetGroupListCommon</include>
-						<include content="WidgetListCategories" condition="Library.HasContent(music)">
+						<include content="WidgetListCategories" condition="Library.HasContent(music) + !Skin.HasSetting(HomeNoCategories)">
```

Comment:
hmm mostly we use all-lowercase with _ as word separator for settings. perhaps change it?